index.html

```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
     </header> 
 
     <section class="landing-page all-body" id="body-cover">
-               
-                <h1 class="title">I'm Mark <br>Glad to see you!</h1>
-                <p class="intro">
-                    I'm a software developer! I can help you build a product , 
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and have a project you need coded, don't 
-                    hestiate to contact me.
-                </p>
-            
-                <p class="lets-connect">LET'S CONNECT</p>
-                    
-                <ul class="social-buttons">
-                    <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
-                </ul>
+               <div class="ctn">
+                    <h1 class="title">I'm Mark <br>Glad to see you!</h1>
+                    <p class="intro">
```

Comment:
- [ ] Kindly center well the header
- Your Design 
<img width="1440" alt="Screenshot 2023-05-12 at 11 10 27" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/9137baad-2c89-4c82-8006-7bc2721bb754">

-  Figma Design 
<img width="1440" alt="Screenshot 2023-05-12 at 11 15 22" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/55772f7e-ed1a-4038-ba2f-bccbd9005198">

Reply:
![Screenshot from 2023-05-13 00-03-05](https://github.com/MarkYegon7/Portfolio-Setup/assets/85193476/5aeae3ab-5470-46bb-90ab-335ebc223498)
This is how my project looks like.

The first reviewer marked another project. That is evident from line 21 which says I'm Mark compared to the screenshot provided by the reviewer.